src/box/lua/console.c

```diff
@@ -326,11 +326,11 @@ lbox_console_readline(struct lua_State *L)
 				rl_on_new_line();
 				rl_replace_line("", 0);
 				lua_pushstring(L, "");
+				lua_pushboolean(L, sigint_called);
```

Comment:
Shouldn't we also adjust `console_push_line()`?

Reply:
Which way?

Reply:
You are right, fixed.

Reply:
Looks resolved now.